src/main/java/net/dv8tion/jda/core/managers/ChannelManager.java

```diff
@@ -367,6 +367,82 @@ public ChannelManager removePermissionOverride(IPermissionHolder permHolder)
         return this;
     }
 
+    /**
+     * Syncs all {@link net.dv8tion.jda.core.entities.PermissionOverride PermissionOverrides} of this Channel with
+     * its parent ({@link net.dv8tion.jda.core.entities.Category Category}).
+     * <br/>After this operation, all {@link net.dv8tion.jda.core.entities.PermissionOverride PermissionOverrides}
+     * will be exactly the same as the ones from the syncSource.
+     * <br/>This behaves as if calling {@link #sync(Channel)} with this Channel's {@link net.dv8tion.jda.core.entities.Channel#getParent() Parent}.
+     *
+     * @throws  java.lang.IllegalStateException
+     *          If this Channel has no parent
+     * @throws  net.dv8tion.jda.core.exceptions.InsufficientPermissionException
+     *          If the currently logged in account does not have {@link net.dv8tion.jda.core.Permission#MANAGE_PERMISSIONS Permission.MANAGE_PERMISSIONS}
+     *          in this channel
+     *
+     * @return  ChannelManager for chaining convenience
+     */
+    @CheckReturnValue
+    public ChannelManager sync()
+    {
+        if(channel.getParent() == null)
+            throw new IllegalStateException("sync() requires a parent category");
+        return sync(channel.getParent());
+    }
+
+    /**
+     * Syncs all {@link net.dv8tion.jda.core.entities.PermissionOverride PermissionOverrides} of this Channel with
+     * the given ({@link net.dv8tion.jda.core.entities.Channel Channel}).
+     * <br/>After this operation, all {@link net.dv8tion.jda.core.entities.PermissionOverride PermissionOverrides}
+     * will be exactly the same as the ones from the syncSource.
+     * <br/>This will only work for Channels of the same {@link net.dv8tion.jda.core.entities.Guild Guild}.
```

Comment:
Add a warning that this will override any previous changes to permission overrides